src/pip/_internal/commands/debug.py

```diff
@@ -62,6 +62,14 @@ def create_vendor_txt_map():
     return dict(line.split('==', 1) for line in lines)  # type: ignore
 
 
+def create_debundle_txt_map():
+    # type: () -> Dict[str, str]
+    wheels = [fn for fn in os.listdir(pip._vendor.WHEEL_DIR)]
+    # Transform into "module" -> version dict.
+    return dict((wheel.split('-')[0],
+                wheel.split('-')[1]) for wheel in wheels)  # type: ignore
```

Comment:
```suggestion
    return dict(sorted(wheel.split("-")[:2] for wheel in wheels))
```

Reply:
Why the `sorted` call? Do we expect the directory to contain multiple files for a given package? I would probably just do

```suggestion
    return dict(wheel.split('-', 2)[:2] for wheel in wheels)
```